src/app/pages/tabs/profile/profile.page.ts

```diff
@@ -40,10 +40,18 @@ export class ProfilePage implements OnInit {
         title: "EDUCATION_QUALIFICATION",
         key: "education_qualification"
       },
+      {
+        title: 'LANGUAGES',
+        key: 'languages',
+      },
       {
         title: "EMAIL_ID",
         key: "emailId"
       },
+      {
+        title: "ORGANIZATION",
+        key: "organizationName"
```

Comment:
u can change this back to organization

Reply:
name conflicts in data so i kept the same ...